### PR TITLE
ZUR-158 My PR adds a feature

### PR DESCRIPTION
**Pull request by odohemma**
# Fixes Issue 

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ZUR-158**

# Changes proposed

## I installed react-split and changed the following files: MessageBoard.jsx, MessagePane.jsx and HoverItems.jsx. The changes I made in the aforementioned files ensure that when a user clicks on the reply icon on the message pane, a sidebar to the right of the message pane opens up.

https://user-images.githubusercontent.com/33616969/202287384-124d7dca-d4b6-4cd6-8d41-8aedb9210873.mp4

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title and description of the PR is clear and explains the approach.
- [ ] I am making a pull request against the **dev branch** (left side).
- [ ] My commit messages styles matches our requested structure.
- [ ] My code additions will fail neither code linting checks nor unit test.
- [ ] I am only making changes to files I was requested to.

# Screenshots/ Videos

https://user-images.githubusercontent.com/33616969/202288172-a617b54e-1331-45f6-a9c8-6c01a46432a7.mp4

### CODE REVIEW

**Comment by JulianaSau** [comment]
A component for threads already exists all you had to do was import it. I gave you the links to where they were

File: package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "dev": "lerna run dev --parallel",
     "postinstall": "lerna bootstrap",
     "format": "lerna run prettify --parallel",
-    "lint": "concurrently yarn:lint:*",
+    "lint": "concurrently F:lint:*",
```
Comment (JulianaSau) [inline package.json:27]: Why edit this?

File: package.json [hunk 2]
```diff
@@ -49,5 +49,10 @@
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
+  },
+  "dependencies": {
+    "9000": "^0.1.3",
+    "kill-port": "^2.0.1",
+    "react-split": "^2.0.14"
```
Comment (JulianaSau) [inline package.json:56]: These are very unnecessary dependencies. Only install dependencies when given explicit permission to do so

File: packages/root-config/src/assets/plugin.importmap.json [hunk 3]
```diff
@@ -14,6 +14,6 @@
     "@zuri/zuri-plugin-tools": "https://externaltools.zuri.chat/zuri-zuri-plugin-tools.js",
     "@zuri/zuri-plugin-channels": "https://channels.zuri.chat/static/main/dist/zuri-zuri-plugin-channels.js",
     "@zuri/zuri-plugin-contributions": "https://ct.zuri.chat/js/app.js",
-    "@zuri/zuri-plugin-messaging": "https://chat.zuri.chat/zuri-messaging.js"
+    "@zuri/zuri-plugin-messaging": "http://localhost:5013/zuri-messaging.js"
```
Comment (JulianaSau) [inline packages/root-config/src/assets/plugin.importmap.json:17]: When you finish coding, you should change it back to how its supposed to be

File: packages/ui/src/message-board/MessageBoard.jsx [hunk 4]
```diff
@@ -82,46 +85,79 @@ function MessageBoard({
...
+          </div>
+        </MessageBoardContainer>
+        {showSidebar && (
+          /* <div style={{ border: "10px solid red" }}>
+          <CommentBoard />
+        </div> */
+
+          /* <MessageBoardContainer
+          style={{ border: "10px solid red" }}
+        ></MessageBoardContainer> */
```
Comment (JulianaSau) [inline packages/ui/src/message-board/MessageBoard.jsx:148]: You should use the comment board component already provided

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (odohemma) [reply package.json:27]: I have no idea how this happened. I will correct it.

File: package.json (hunk 2 above)
Comment (odohemma) [reply package.json:56]: For reasons I don't know, unless I kill the port, running yarn dev always return an error message. 
See the attached screenshot. I will remove the "9000" and "kill-port" dependencies.

The "react-split" dependency on the other hand, was used in solving the task at hand.

Screenshot 2022-11-14 101202

File: packages/root-config/src/assets/plugin.importmap.json (hunk 3 above)
Comment (odohemma) [reply packages/root-config/src/assets/plugin.importmap.json:17]: Alright. I will correct it.

File: packages/ui/src/message-board/MessageBoard.jsx (hunk 4 above)
Comment (odohemma) [reply packages/ui/src/message-board/MessageBoard.jsx:148]: The comment board component wasn't rendering anything, I used another component to be sure the feature I created works.

**Pull request closed without merging**